openreview/conference/helpers.py

```diff
@@ -201,6 +201,11 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
     builder.set_review_stage(get_review_stage(note))
     builder.set_ethics_review_stage(get_ethics_review_stage(note))
 
+    decision_heading_map = note.content.get('home_page_tab_names')
```

Comment:
are you sure this work? becauase `get_result` is going to call set_home_page: https://github.com/openreview/openreview-py/blob/master/openreview/conference/builder.py#L2782

Could you add a new property in the conference call that store the `decision_heading_map` and inside of the `set_home_page` implementation, you can check if that map is present then set the decisions layout otherwise set the tab layout.

Reply:
Sorry I just realized that, the changes are great!